README.md

```diff
@@ -22,20 +22,20 @@ Step-by-step Deep Learning Tutorials on Apache Spark using [BigDL](https://githu
 ### Environment
 + Python 2.7
 + JDK 8
-+ Apache Spark 2.2.0
++ Apache Spark >= 2.2.1
 + Jupyter Notebook 4.1
-+ BigDL 0.5.0
++ BigDL 0.7.0
 + [Setup env on Mac OS](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupMac.md) / [Setup env on Linux](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupLinux.md)
 
 ### Start Jupyter Server
-* Run ```pip install BigDL==0.5.0```
-* Run ```jupyter notebook --notebook-dir=./ --ip=* --no-browser```
+* Run ```sudo pip install BigDL==0.7.0```
```

Comment:
I think we should not use sudo to install the packages under `/usr/local/`. Could we use the `PYTHONPATH` to handle privileges ?

Reply:
Yes. Accept it.
I will remove 'sudo'